app/src/main/java/app/revanced/integrations/returnyoutubedislike/ReturnYouTubeDislike.java

```diff
@@ -365,6 +350,81 @@ public void updateDrawState(TextPaint tp) {
         return true;
     }
 
+    private static boolean segmentedValuesSet = false;
+    private static float segmentedVerticalShiftRatio;
+    private static float segmentedLeftSeparatorFontRatio;
+    private static float segmentedLeftSeparatorHorizontalScaleRatio;
+
+    /**
+     * Set the segmented adjustment values, based on the device.
+     */
+    private static void setSegmentedAdjustmentValues() {
+        if (segmentedValuesSet) {
+            return;
+        }
+
+        String deviceManufacturer = Build.MANUFACTURER;
+        final int deviceSdkVersion = Build.VERSION.SDK_INT;
+        DisplayMetrics metric = ReVancedUtils.getContext().getResources().getDisplayMetrics();
+        final float deviceScaledDensity = metric.scaledDensity;
+        final int deviceDensityDpi = metric.densityDpi;
+        final int deviceWidth = metric.widthPixels;
+        final int deviceHeight = metric.heightPixels;
+        LogHelper.printDebug(() -> "Device manufacturer: '" + deviceManufacturer + "' SDK: " + deviceSdkVersion
+                + " scaledDensity: " + deviceScaledDensity + " densityDpi: " + deviceDensityDpi
+                + " width: " + deviceWidth + " height: " + deviceHeight);
+
+        // device used to determine the configuration below
+        final String configManufacturer; // very important
+        final int configSdk; // important
+        final float configScaledDensityDpi; // important only if using non default system font size
+        final int configDensityDpi, configWidth, configHeight; // rarely important
```

Comment:
These are not used anywhere except for this method. What is the purpose of calling `setSegmentedAdjustmentValues()` if it does not change anything?

Reply:
The config variables are for documentation, to show what device was used for the variables segmentedVerticalShiftRatio, segmentedLeftSeparatorFontRatio, segmentedLeftSeparatorHorizontalScaleRatio.

The config variable documentation could be left out, but then it's a mystery what device was used to come up with the settings.

Reply:
To simplify this, I removed all the config options except the manufacturer and android build version.